packages/graph/hash_graph/tests/integration/postgres/data_type.rs

```diff
@@ -0,0 +1,54 @@
+use crate::postgres::Database;
+
+#[test]
```

Comment:
Can't we just use `tokio::test` here and avoid the runtime stuff? Or is that what you were talking about with `drop`?

Reply:
Can you put a comment saying that (where we create `rt`)

Reply:
Added a coment here: d074238616ea5cb3fe3368feb4d1aedac0af6455